docs/docs/02-configuration/11-queryable-projections.mdx

```diff
@@ -0,0 +1,73 @@
+import Tabs from '@theme/Tabs';
+
+# IQueryable projections
+
+Mapperly does support `IQueryable<T>` projections:
+
+<!-- do not indent this, it won't work, https://stackoverflow.com/a/67579641/3302887 -->
+
+<Tabs>
+<TabItem value="definition" label="Mapper definition">
+
+```csharp
+[Mapper]
+public static partial class CarMapper
+{
+    // highlight-start
+    public static partial IQueryable<CarDto> ProjectToDto(this IQueryable<Car> q);
+    // highlight-end
+}
+```
+
+</TabItem>
+<TabItem value="usage" label="Usage">
+
+```csharp
+var dtos = await DbContext.Cars
+    .Where(...)
+    // highlight-start
+    .ProjectToDto()
+    // highlight-end
+    .ToListAsync();
+```
+
+</TabItem>
+</Tabs>
+
+This is useful in combination with EntityFramework and other ORM solutions
```

Comment:
Would change this to
> This is useful in combination with Entity Framework and other ORM solutions
which expose `IQueryable<T>`. Only fields present in the target class will be retrieved from the database.

The last sentence (code inlining) is not interesting to Mapperly consumers IMO, which is why I would remove it

Reply:
Improved.